packages/merge-conflicts/src/browser/merge-conflicts-decorations.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from "inversify";
+import { EditorDecorationsService, OverviewRulerLane, Range } from "@theia/editor/lib/browser";
+import { MergeConflictUpdateParams } from "./merge-conflicts-service";
+import URI from "@theia/core/lib/common/uri";
+
+enum Type {
```

Comment:
please either put it under `MergeConflictsDecorations` namespace or reexport and name `MergeConflictsDecorationType`